### ISSUE TITLE
Errors after upgrading to v1.0.0 

### ISSUE THREAD

**Issue by rejas** [tatsuyafw/gulp-nightwatch#55]
No problems with running this plugin in v0.3.3 but updateing to v1.0.0 this comes up when calling:

error:
```
[14:34:29] Starting nightwatch...

[Ilb2] Test Suite
=================
    POST  /wd/hub/session - ECONNREFUSED
Error: connect ECONNREFUSED 127.0.0.1:4444
   An occurred error while retrieving a new session: "connect ECONNREFUSED 127.0.0.1:4444"
       at Socket.socketErrorListener (_http_client.js:387:9)
_________________________________________________

TEST FAILURE: 1 error during execution 0 tests failed, 0 passed. 1.218s

 × ilb2

  An occurred error while retrieving a new session: "connect ECONNREFUSED 127.0.0.1:4444"
       at Socket.socketErrorListener (_http_client.js:387:9)

   SKIPPED:
   - Open and Close lightbox page
   - Caption
   - Deep links
   - Dynamic add
   - Manual trigger
   - Navigation

[14:34:31] '<anonymous>' errored after 2.18 s
[14:34:31] Error in plugin "gulp-nightwatch"
Message:
    nightwatch exited with code 5
Details:
    domainEmitter: [object Object]
    domain: [object Object]
    domainThrown: false

[14:34:31] 'night:js' errored after 3.32 s
```

my config:
```
{
    "src_folders" : [ "./test/src" ],
    "output_folder" : "./test/reports",

    "selenium" : {
        "start_process" : true,
        "server_path" : "./test/resources/selenium-server-standalone-3.13.0.jar",
        "log_path" : false,
        "port" : 4444,
        "cli_args" : {
            "webdriver.chrome.driver" : "./test/resources/chromedriver.exe"
        }
    },

    "test_settings" : {
        "default" : {
            "launch_url" : "http://localhost",
            "selenium_port"  : 4444,
            "selenium_host"  : "localhost",
            "silent": true,
            "desiredCapabilities": {
                "browserName" : "chrome",
                "javascriptEnabled" : true,
                "acceptSslCerts" : true
            }
        }
    }
}
```

**Comment by tatsuyafw** [comment]
@rejas 

As far as I saw the logs you pasted, a selenium server dose not seem to be running in your environment. Please access to `http://localhost:4444` with a browser, and check whether a selenium server works fine.

**Comment by rejas** [comment]
Isnt gulp-nightwatch starting the selenium server itself? My gulp workflow only starts a dev server at http://localhost:8080/docs/

**Comment by tatsuyafw** [comment]
> Isnt gulp-nightwatch starting the selenium server itself?

Yes. To be exact, gulp-nightwatch starts a selenium server via nightwatch when `start_process` set to true in nightwatch.js.
But, I've found that gulp-nightwatch v1.0.0 does not start a selenium server. Sorry, this is a bug.  I'll fix it.

**Comment by rejas** [comment]
Glad to know its a bug and not my incompetence for once :-)

**Comment by tatsuyafw** [comment]
@rejas 

I've fixed this issue, and released as v1.0.1.

**Comment by rejas** [comment]
@tatsuyafw Works! Thx a lot!
